src/de/mod.rs

```diff
@@ -1867,6 +1867,7 @@ macro_rules! deserialize_primitives {
         }
 
         /// Character represented as [strings](#method.deserialize_str).
+        #[inline]
```

Comment:
Have you actually found this to make a difference?

The primary purpose of `#[inline]` is to allow inlining across compilation units.  Within a compilation unit, the compiler is just going to decide on its own and mostly ignore this attribute unless you say `#[inline(always)]`.

If these functions are not going to be called from any external library, I doubt it would make a difference.

Reply:
No, added just for consistence. Actually I want to run some profiler one day, just need time to learn how to do that and find a which one that could be run on Windows. Most tools supports only Linux. `cargo flamegraph` looks promising.